cmd/controller/main.go

```diff
@@ -231,6 +233,24 @@ func main2() error {
 
 	go controller.Run(stop)
 
+	if *additionalNamespaces != "" {
```

Comment:
Not sure about this. Unless I'm missing something, there will already be an informer created for all namespaces and then we are creating a new one for the additional namespaces.

Reply:
Would that cause issues? @ThomasVitt could you please have a look?